packages/flutter/lib/src/widgets/text.dart

```diff
@@ -1401,3 +1497,32 @@ class _SelectableTextContainerDelegate extends MultiSelectableSelectionContainer
     super.didChangeSelectables();
   }
 }
+
+class _TextSpanContentController extends SelectedContentController<TextSpan> {
+  _TextSpanContentController(super.value);
+
+  @override
+  TextSpan buildContents() {
```

Comment:
What is buildContents supposed to be used for? Maybe include an example that overrides it?

Reply:
I removed it, I think I was imagining something similar to `TextEditingController.buildTextSpan` when I first wrote this but it's not needed.